Mathlib/RingTheory/RingHom/Integral.lean

```diff
@@ -30,11 +30,10 @@ theorem isIntegral_respectsIso : RespectsIso fun f => f.IsIntegral := by
 
 theorem isIntegral_isStableUnderBaseChange : IsStableUnderBaseChange fun f => f.IsIntegral := by
   refine IsStableUnderBaseChange.mk isIntegral_respectsIso ?_
-  introv h x
-  refine TensorProduct.induction_on x ?_ ?_ ?_
-  · apply isIntegral_zero
-  · intro x y; exact IsIntegral.tmul x (h y)
-  · intro x y hx hy; exact IsIntegral.add hx hy
+  introv int
+  change (algebraMap _ _).IsIntegral
```

Comment:
Probably
```suggestion
  rw [← algebraMap_isIntegral_iff]
```
(untested)

Reply:
@Ruben-VandeVelde the issue is actually fixed in #28225, where `IsStableUnderBaseChange.mk` was using `Algebra.TensorProduct.includeLeftRingHom`, a ring hom with less API than `algebraMap S (S ⊗[R] T)`.

Reply:
that is merged, so this is fixed